testsuite/MDAnalysisTests/core/test_topologyobjects.py

```diff
@@ -22,9 +22,10 @@
     assert_raises,
 )
 
-import MDAnalysis
+import MDAnalysis as mda
 from MDAnalysis.core.topologyobjects import (
     TopologyGroup, TopologyObject, TopologyDict,
+    # TODO: the following items are not used
     Bond, Angle, Dihedral, ImproperDihedral,
```

Comment:
I never used those properties myself. It would be nice if some else would write tests for them.